### PR TITLE
docs(rollup): explain that the stamp variable is magically replaced

### PR DESCRIPTION
**Pull request by alexeagle**

### CODE REVIEW

**Review by jbedard — APPROVED** [review]

File: packages/rollup/index.bzl [hunk 1]
```diff
@@ -143,6 +143,7 @@ function parseStatusFile(p) {
   return results;
 }
 
+// This undefined variable will be replaced with the full path during the build.
```
Comment (jbedard) [inline packages/rollup/index.bzl:146]: Could maybe say "the undefined 'bazel_version_file' variable" to be a bit more clear 🤷

File: packages/rollup/index.js [hunk 2]
```diff
@@ -239,6 +239,6 @@ async function main(args) {
   }
 }
 
-if (require.main === module) {
-  main(process.argv.slice(2)).then(r => (process.exitCode = r));
-}
+if (require.main == module) {
+  (async () => process.exitCode = await main(process.argv.slice(2)))();
```
Comment (jbedard) [inline packages/rollup/index.js:243]: Does this cause any actual change? How about the `==` change above?

Comment (alexeagle) [reply packages/rollup/index.js:243]: no, just style to match other files while I was here

File: packages/rollup/index.bzl (hunk 1 above)
Comment (alexeagle) [reply packages/rollup/index.bzl:146]: that's a good idea but I don't want to wait for CI again. Maybe I should make CI faster? :)

Comment (jbedard) [reply packages/rollup/index.bzl:146]: yes you should :P

**Pull request merged**